### ISSUE TITLE
Add release apk

### ISSUE THREAD

**Issue by linwei5d** [alexbakker/webdav-provider#22]
can you add a release apk file so that I can use this project without building it？

**Comment by liqlos** [comment]
that would be great

**Comment by ippocratis** [comment]
Building is not that hard
If anyone have some sort of trouble building this project I have builded and uploaded the apk to a [telegram channel](https://t.me/s/ippossackofhacks/604)

Also I uploaded [here](https://t.me/s/favandroidapps/6) the apk with client certificate mutual TLS authentication from defkev [repo](https://github.com/defkev/webdav-provider/tree/patch-mtls)
There is an open [pull request](https://github.com/alexbakker/webdav-provider/pull/21) 7 months now

**Comment by stokito** [comment]
Please publish it to https://f-droid.org/

I also added the app to [Awesome WebDAV list](https://github.com/WebDAVDevs/awesome-webdav/blob/main/readme.md#android)

**Comment by i30817** [comment]
I think I need a more recent build. When I tried to add with the build given in the telegram channel, it started printing

‘Couldnt establish webdav connection: expected leading [0-9a-fA-F] character but was 0x1f‘

If I try to use curl with the urls for the sever it prints the html of the page you get on localhost:8008 as expected.


I'm using python webdav installed with 

pip install PyWebDAV3 --break-system-packages

And run with 

davserver -D /media/ -n -H 0.0.0.0

Before listening on all hostnames I failed connecting, you probably should put the 0.0.0.0 trick on the readme.

I can't build because my PC only has 2gb of memory and a few gib of free / space too (it's a eeepc).

Edit: started using RSAF and apache webdav instead.
